recipes/jupyterlab-system-monitor/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "jupyterlab-system-monitor" %}
+{% set version = "0.7.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/jupyterlab-system-monitor-{{ version }}.tar.gz
+  sha256: 726ee949746da1aa27eb494e1da0e3d40508ddd97aad3d2e28243f17400d28fe
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - jupyter-packaging
+    - pip
+    - python >=3.6
+  run:
+    - jupyterlab >=3.0.0rc10,==3.*
+    - jupyterlab-topbar >=0.6.0,<0.7.0
+    - nbresuse >=0.4.0,<0.5.0
```

Comment:
`0.8.0` now depends on `jupyter-resource-usage >=0.5.0,<0.6.0` instead of `nbresuse`.